configs/config.ci.yaml

```
@@ -80,3 +80,6 @@ app:
   # main secret for various MAC and encryptions like domain registration
   # token and encrypted private JWKs. "random" generates an ephemeral secret.
   secret: random
+  # Enable/Disable RBAC verification
+  # TODO Change to true when the feature is ready
```

Let's include the ticket number, so it is easier to find the places that have to be touched later.

```suggestion
  # TODO remove override when HMS-3521 is implemented
```